pkg/operators/ebpf/ebpf.go

```diff
@@ -56,6 +56,8 @@ const (
 	ParamTraceKernel = "trace-pipe"
 
 	kernelTypesVar = "kernelTypes"
+
+	BPF_TRACE_UPROBE_MULTI ebpf.AttachType = 48
```

Comment:
Could you add a comment explaining where this value comes from?

The constant is declared in cilium/ebpf but it is internal:
https://github.com/cilium/ebpf/blob/v0.15.0/internal/sys/types.go#L67
It is part of the Linux API:
https://github.com/torvalds/linux/blob/v6.8/include/uapi/linux/bpf.h#L1049